src/server/routes/controllers/utilities/responseManager.js

```diff
@@ -0,0 +1,25 @@
+
+const CODES = {
+	successes : {
+		OK : [200, 'OK']
+	},
+	errors : {
+		BAD_REQUEST : [400, 'BAD_REQUEST'],
+		UNAUTHORIZED : [401, 'UNAUTHORIZED'],
+		INTERNAL_ERROR : [500, 'INTERNAL SERVER ERROR']
+	}
+}
+
+function createResponse(res, code, extraPayload) {
+	const payload = {
+		description: code[1],
```

Comment:
Let's make every code an object, and not use arrays indexing to extract values. It's very unclear.

Reply:
ok